matrix-sdk-android/src/main/java/org/matrix/android/sdk/api/auth/data/IdentityProvider.kt

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright 2020 The Matrix.org Foundation C.I.C.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.matrix.android.sdk.api.auth.data
+
+import android.os.Parcelable
+import com.squareup.moshi.Json
+import com.squareup.moshi.JsonClass
+import kotlinx.android.parcel.Parcelize
+
+@JsonClass(generateAdapter = true)
+@Parcelize
+data class IdentityProvider(
```

Comment:
I suspect it could be a confusing name because of the Identity Service. Maybe rename to `SsoParameters` or `SsoDetails`?

Reply:
Renamed to `SsoIdentityProvider`